examples/docs_beta_snippets/docs_beta_snippets/guides/external-systems/assets-with-io-managers.py

```diff
@@ -12,15 +12,15 @@ def raw_sales_data() -> pd.DataFrame:
 
 
 # highlight-start
-@dg.asset
-# Load the upstream `raw_sales_data` asset as input & specify the returned data type (`pd.DataFrame`)
+@dg.asset(deps=[raw_sales_data])
+# Load the upstream `raw_sales_data` asset as an input & specify the returned data type (`pd.DataFrame`)
 def clean_sales_data(raw_sales_data: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Doesn't make sense to me to double-encode the deps...

Reply:
Did you read description yet? The crux of it is, the IO manager docs and the graph-backed asset docs are now the only places where we front-load the input syntactic sugar, which to me is weird. I think when introducing IO managers we should do so without also introducing the syntactic sugar (or at least, introduce it separately).

Maybe that's an argument for also setting the ins parameter to explicitly map the asset key to the input name.